backend/schema/room.py

```diff
@@ -231,3 +231,9 @@ def is_object_id(cls, values):
             )
 
         return values
+
+
+class AddToRoom(BaseModel):
```

Comment:
Nice work.
Do you think it's possible in fastapi it's possible to add a body parameter in the endpoint function without creating a class for it, so that rather than creating a class with a single instance variable would be avoided ?

Reply:
yeah absolutely I think that is possible. I will effect the change.

Reply:
After trying different ways to add the input data as a body param, IMO using the class was the best way to keep the code clean and the function less complex.

Reply:
Alright thanks